Source/Core/Common/Common.vcxproj

```diff
@@ -175,6 +176,7 @@
     <ClCompile Include="Hash.cpp" />
     <ClCompile Include="IniFile.cpp" />
     <ClCompile Include="JitRegister.cpp" />
+    <ClCompile Include="LdrWatcher.cpp" />
```

Comment:
Well of course. Preferably using `#ifdef WIN32` around the whole file to make sure no extra logic is added in the already complex build system.